AzureBasicLoadBalancerUpgrade/module/AzureBasicLoadBalancerUpgrade/modules/ValidateScenario/ValidateScenario.psm1

```diff
@@ -68,7 +69,7 @@ function _GetScenarioBackendType {
         $backendType = 'VMSS'
     }
     ElseIf ([string]::IsNullOrEmpty($backendMemberTypes[0])) {
-        log -Message "[Test-SupportedMigrationScenario] Basic Load Balancer backend pools are empty"
+        log -ErrorAction Stop -Severity 'Error' -Message "[Test-SupportedMigrationScenario] Basic Load Balancer backend pools are empty"
```

Comment:
@cbellee We're supporting empty backends for migration - this was a customer ask and decreases the unsupported scenarios for customers to review.

Reply:
I fixed the code failing that was failing the tests but obviously the tests weren't correct in the first place. I'll revert the change & modify the test to pass if backends are empty.